test/xbgpu/test_bsend.py

```diff
@@ -62,6 +62,7 @@ def sensors(outputs: Sequence[BOutput]) -> SensorSet:
     sensors = SensorSet()
     for output in outputs:
         sensors.add(Sensor(int, f"{output.name}.beng-clip-cnt", "Number of output samples that are saturated."))
+        sensors.add(Sensor(int, f"{output.name}.tx.next-timestamp", "Timestamp that has not yet been sent"))
```

Comment:
Aha, the BPipeline updates this in BSend. Whereas the XPipeline updates the corresponding sensor in the sender-loop.

Nice one.

Reply:
Yeah. Given unlimited time, I'd like to move more of that logic into XSend, so that it's easier to test in isolation.